packages/foam-workspace-manager/test/utils/utils.test.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  parseNoteTitleFromMarkdown,
+  parseNoteLinksFromMarkdown,
+} from '../../src/utils/utils';
+
+const pageA = `
+# Page A
+
+## Section
+- [[page-b]]
+- [[page-c]]
+`;
+
+const pageB = `
+# Page B
+`;
+
+const pageC = `
+foo
+bar
+`;
+
+describe('WorkspaceManager', () => {
+  it('finds top level headings', () => {
+    const titleA = parseNoteTitleFromMarkdown(pageA);
+    const titleB = parseNoteTitleFromMarkdown(pageB);
+    const titleC = parseNoteTitleFromMarkdown(pageC);
+
```

Comment:
might be worth adding a test for a page that has multiple top level headings?

```
# Page D
hello world

# Another header
hello world
```

Reply:
Thanks for the suggestion, done :)